CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ have notable changes.
 
 Changes since v2.33
 
+### Changes
+- "Show related events" has been removed from application page.
+- Sensitive events (currently `#{:application.event/remarked :application.event/attachments-redacted}`) now show eye-open/eye-closed icon when `:application/public` is true/false. This change is only visible to handler, reviewer and decider. (#3156)
```

Comment:
We could try to write this changelog for users as it's now rather technical.

Reply:
Maybe just:
> The handling users can now see whether an event is shown to the applicant from a small eye icon in the event history. (#3156)

I think it would be clearer to show this for all events, not just "sensitive events".